### ISSUE TITLE
xpra not found on Ubuntu 19.10

### ISSUE THREAD

**Issue by AndreiRegiani** [bkerler/ghidra_installer#6]
```
➜  ghidra_installer git:(master) ./ghidra4K
./run_scaled: line 75: xpra: command not found
./run_scaled: line 77: xpra: command not found
./run_scaled: line 77: xpra: command not found
```

`xpra` dependency doesn't seem to be available on apt-get.

**Comment by bkerler** [comment]
Yes, xpra will break things. I haven't yet found a replacement for it.

**Comment by AndreiRegiani** [comment]
It seems `xpra` is really NOT needed, to force Ghidra to scale up you can edit: `ghidra/support/launch.properties`

```
VMARGS_LINUX=-Dsun.java2d.uiScale=1
```
`2` is working well on 4K for me.

**Comment by bkerler** [comment]
Fixed in commit 9228e29. xpra isn't used anymore.
